packages/common/src/projects/_internal/getProjectEditorConfigOptions.ts

```diff
@@ -0,0 +1,82 @@
+import { IArcGISContext } from "../../ArcGISContext";
+
+import { getCategoryItems } from "../../core/schemas/internal/getCategoryItems";
+import { getFeaturedContentCatalogs } from "../../core/schemas/internal/getFeaturedContentCatalogs";
+import { getFeaturedImageUrl } from "../../core/schemas/internal/getFeaturedImageUrl";
+import { getLocationExtent } from "../../core/schemas/internal/getLocationExtent";
+import { getLocationOptions } from "../../core/schemas/internal/getLocationOptions";
+import { getTagItems } from "../../core/schemas/internal/getTagItems";
+import { groupsToComboBoxItems } from "../../core/schemas/internal/groupsToComboBoxItems";
+import { UiSchemaElementOptions } from "../../core/schemas/types";
+import { IHubProject } from "../../core/types/IHubProject";
+import { checkPermission } from "../../permissions/checkPermission";
+
+/**
+ * Construct the Project Editor configuration options
+ * @param entity
+ * @param context
+ * @returns
+ */
+export async function getProjectEditorConfigOptions(
```

Comment:
Similar to what we just did with creating a base `IHubItemEntitySchema` that all entities can spread, I think it might be nice to create a `baseItemEntityConfigOptions` array that all item entities can spread because I don't think any of this is necessarily project-specific.

Reply:
Let's revisit this on a subsequent PR / issue